apps/passport-server/src/database/queries/telegram/fetchTelegramEvent.ts

```diff
@@ -340,3 +341,35 @@ export async function fetchTelegramAnonMessagesWithTopicByNullifier(
   );
   return result.rows;
 }
+
+export async function fetchUserBySemaphoreId(
+  client: Pool,
+  semaphoreId: string
+): Promise<TelegramConversation> {
+  const result = await sqlQuery(
+    client,
+    `
+    SELECT *
+    FROM telegram_bot_conversations 
+    WHERE semaphore_id = $1
+    `,
+    [semaphoreId]
+  );
+  return result.rows[0] ?? null;
+}
+
+export async function fetchUserByTelegramId(
```

Comment:
Rename this function to something that isn't just "user" - user means something else in this codebase.